### DISCUSSION TITLE
minRegionArea not taken into account

### DISCUSSION THREAD

**Discussion by niko-dellic** [isaac-mason/recast-navigation-js#432] [General]
regardless of the minRegionArea value, no regions are culled from the mesh. Looking for anyone else experiencing the same thing or potential workarounds. There are citations of this being an issue in other deployments of recast, such as in [unreal](https://forums.unrealengine.com/t/the-navigation-mesh-build-appears-to-ignore-the-min-region-area-setting/360487/6)

**Comment by isaac-mason** [comment]
Hey!

If you're hoping to use minRegionArea to cull disconnected parts of a navmesh, e.g. areas on top of walls or ceilings etc, you might find the flood fill pruning example interesting:

https://recast-navigation-js.isaacmason.com/?path=/story/advanced-flood-fill-pruning--flood-fill-pruning

https://github.com/isaac-mason/recast-navigation-js/blob/main/packages/recast-navigation/.storybook/stories/advanced/flood-fill-pruning.stories.tsx
